bazel/BUILD.bazel

```diff
@@ -1,13 +1,30 @@
-package(default_visibility = ["//visibility:public"])
+load("//tools:defaults.bzl", "pkg_npm", "ts_library")
+
+
+ts_library(
+    name = "lib",
+    srcs = [],
+    visibility = [
+        "//:__subpackages__",
+    ],
+    deps = [
+        "//bazel/benchmark/driver-utilities",
+    ]
+)
 
 filegroup(
-    name = "files",
+    name = "npm_files",
+    visibility = [
+        "//:__subpackages__",
+    ],
     srcs = [
         "BUILD.bazel",
         "expand_template.bzl",
         "extract_js_module_output.bzl",
-        "//dev-infra/bazel/api-golden:files",
-        "//dev-infra/bazel/browsers:files",
-        "//dev-infra/bazel/remote-execution:files",
+        "index.bzl",
```

Comment:
I think the `index.bzl` file needs to go into the project root as otherwise `rules_nodejs` will not consider it as a package that can be loaded from within Bazel.